modules/vsphere-problem-detector-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator_{context}"]
+= vSphere Problem Detector Operator
+
+[discrete]
+== Purpose
+
+The {operator-name} checks the cluster for the most common installation and misconfiguration issues.
```

Comment:
It might be useful to explicitly state that the checks are carried out on OCP installations run on vSphere (and Red Hat OpenStack Platform?) only.